src/main/java/appeng/client/gui/AEBaseGui.java

```diff
@@ -933,4 +927,29 @@ public void func_146977_a( final Slot s )
 	{
 		this.drawSlot( s );
 	}
+
+	protected GuiScrollbar getScrollBar()
```

Comment:
since it is not `final`, subclasses can override this method. This method is often called in the constructor which can be potentially dangerous